0-Parameterised_reports.R

```diff
@@ -15,6 +15,64 @@ updates %>%
   as.list() %>%
   list2env(globalenv())
 
+# Test Raw Data  ==============================================================
+
+raw_data <- Filter(function(x) is(x, "data.frame"), mget(ls()))
+
+expected_file_names <-  c("updates",

```

Comment:
> * If a test fails it would be great to have a message saying where the problem is. I think we've discussed this - so you might already be thinking about it.

So it says the sheet name for all failed tests except the check_expected_sheet_names which I will add in today. Do you want it to be more specific?

Reply:
Yes it might be worth being more specific with the test for blank data. Besides that - this was more something to think about for future tests. I think some datasets might be rather large, or we might have a domain-specific test. In those cases it would help to know which local authority/year/etc is suspect.